pkg/kubectl/.import-restrictions

```diff
@@ -1,151 +1,130 @@
+        "k8s.io/kubernetes/pkg/generated",
+        "k8s.io/kubernetes/pkg/kubectl",
+        "k8s.io/kubernetes/pkg/kubectl/apply",
+        "k8s.io/kubernetes/pkg/kubectl/apply/parse",
+        "k8s.io/kubernetes/pkg/kubectl/apply/strategy",
+        "k8s.io/kubernetes/pkg/kubectl/apps",
+        "k8s.io/kubernetes/pkg/kubectl/cmd",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/auth",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/config",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/create",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/get",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/rollout",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/scalejob",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/set",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/set/env",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/templates",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/testdata/edit",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/testing",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util/editor",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util/editor/crlf",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi/testing",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi/validation",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/util/sanity",
+        "k8s.io/kubernetes/pkg/kubectl/cmd/wait",
+        "k8s.io/kubernetes/pkg/kubectl/explain",
+        "k8s.io/kubernetes/pkg/kubectl/metricsutil",
+        "k8s.io/kubernetes/pkg/kubectl/polymorphichelpers",
+        "k8s.io/kubernetes/pkg/kubectl/proxy",
+        "k8s.io/kubernetes/pkg/kubectl/scheme",
+        "k8s.io/kubernetes/pkg/kubectl/util",
+        "k8s.io/kubernetes/pkg/kubectl/util/hash",
+        "k8s.io/kubernetes/pkg/kubectl/util/i18n",
+        "k8s.io/kubernetes/pkg/kubectl/util/logs",
+        "k8s.io/kubernetes/pkg/kubectl/util/slice",
+        "k8s.io/kubernetes/pkg/kubectl/util/term",
+        "k8s.io/kubernetes/pkg/kubectl/validation",
+        "k8s.io/kubernetes/pkg/printers",
+        "k8s.io/kubernetes/pkg/printers/internalversion",
+        "k8s.io/kubernetes/pkg/registry/rbac/reconciliation",
+        "k8s.io/kubernetes/pkg/util/interrupt",
+        "k8s.io/kubernetes/pkg/util/taints",
+        "k8s.io/kubernetes/pkg/version",
+        "k8s.io/kubernetes/pkg/kubelet/apis",
+        "k8s.io/kubernetes/pkg/master/ports",
+        "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util",
+        "k8s.io/kubernetes/pkg/scheduler/api",
+        "k8s.io/kubernetes/pkg/scheduler/cache",
+        "k8s.io/kubernetes/pkg/scheduler/util",
+        "k8s.io/kubernetes/pkg/security/apparmor",
+        "k8s.io/kubernetes/pkg/util/labels",
+        "k8s.io/kubernetes/pkg/util/node",
+        "k8s.io/kubernetes/pkg/util/slice",
+        "k8s.io/kubernetes/pkg/features",
+        "k8s.io/kubernetes/pkg/registry/rbac/validation",
+        "k8s.io/kubernetes/pkg/scheduler/algorithm",
```

Comment:
Hi @saravanan30erd,
Just trying to understand, there are lots of packages like -
```
k8s.io/kubernetes/pkg/scheduler/*
k8s.io/kubernetes/pkg/util/node
k8s.io/kubernetes/pkg/util/slice
k8s.io/kubernetes/pkg/features
k8s.io/kubernetes/pkg/registry/rbac/validation, etc
```
that are listed under `AllowedPrefixes` but I don't think they are imported anywhere under  `pkg/kubectl`, any specific reason why they are kept or am I missing something?

Reply:
Aah, that was due to transitive dependencies, that was obvious. Sorry for the confusion.